libpod/container_internal.go

```diff
@@ -91,6 +95,10 @@ func (c *Container) rootFsSize() (int64, error) {
 
 // rwSize Gets the size of the mutable top layer of the container.
 func (c *Container) rwSize() (int64, error) {
```

Comment:
For one of these (`rootFsSize()` and `rwSize()`) we should probably compute the size of the user-specified rootfs so we can report something meaningful via Inspect.